<!-- BEGIN KEY -->
public class javax.activity.InvalidActivityException extends java.rmi.RemoteException
<!-- END KEY -->
<!-- BEGIN DATA -->
任何访问 Activity 上下文的方法都可能抛出此异常，它指示尝试调用或与尝试调用相关的 Activity 上下文与 Activity 的当前状态不兼容。如果在禁用 Activity 上下文的方法上接收到 Activity 上下文，则某个容器也可能抛出此异常。此异常通过 org.omg.CORBA.INVALID_ACTIVITY 系统异常跨 J2EE Activity Service ORB 边界进行传播。应用程序应通过试图结束 Activity 来处理此错误。

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.InvalidActivityException()
<!-- END KEY -->
<!-- BEGIN DATA -->
构造一个新实例，其详细消息为 null。

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.InvalidActivityException(String message)
<!-- END KEY -->
<!-- BEGIN DATA -->
构造带有指定详细消息的新实例。

@param   message   详细消息。

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.InvalidActivityException(Throwable cause)
<!-- END KEY -->
<!-- BEGIN DATA -->
构造带有指定 cause 的新 throwable。

@param cause <code>Throwable</code> 类型的链异常

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.InvalidActivityException(String message, Throwable cause)
<!-- END KEY -->
<!-- BEGIN DATA -->
构造带有指定详细消息和 cause 的新 throwable。

@param   message   详细消息。

@param cause <code>Throwable</code> 类型的链异常

<!-- END DATA -->
